src/templates/Layout.jsx

```diff
@@ -8,9 +8,9 @@ export const Layout = ({ children }) => {
 
   useEffect(() => {
     const fetchData = async () => {
-      const data = await getCodeItInfo(`sample/user`);
-      const { email, profileImageSource } = data || {};
-      const profile = data ? { email, profileImageSource } : null;
+      const { data } = await getCodeItInfo(`users/1`);
+      const { email, image_source } = data[0] || {};
+      const profile = data ? { email, image_source } : null;
```

Comment:
비슷한 작업을 두번 하고 계신것 같아요
제가 코드를 보고 이해하기로는 data[0]가 있을 때, 
email과 image_source 값을 profile state에 넣고 싶으신거 같아요

```suggestion
     if (data.legnth === 0) return;
     const {email, image_source} = data[0];
     setProfile({email, image_source});
```

그래서 이렇게 작성하면 코드를 이해하는데 더 수월할 것 같습니다.